open_ce/build_image.py

```diff
@@ -36,29 +36,28 @@
 RUNTIME_IMAGE_PATH = os.path.join(OPEN_CE_PATH, "images",
                                   RUNTIME_IMAGE_NAME)
 REPO_NAME = "open-ce"
-IMAGE_NAME = "open-ce-" + open_ce_version
 TEMP_FILES = "temp_dir"
 
 OPENCE_USER = "opence"
 LOCAL_CONDA_CHANNEL_IN_IMG = "opence-local-conda-channel"
 TARGET_DIR = "/home/{}/{}".format(OPENCE_USER, LOCAL_CONDA_CHANNEL_IN_IMG)
 
-def build_image(local_conda_channel, conda_env_file, container_tool, container_build_args=""):
+def build_image(local_conda_channel, conda_env_file, container_tool, image_version, container_build_args=""):
     """
     Build a container image from the Dockerfile in RUNTIME_IMAGE_PATH.
     Returns a result code and the name of the new image.
     """
     variant = os.path.splitext(conda_env_file)[0].replace(utils.CONDA_ENV_FILENAME_PREFIX, "", 1)
     variant = variant.replace("-runtime", "")
-
+    image_name = REPO_NAME + ":" + image_version + "-" + variant
+    print("Image name: ", image_name)
```

Comment:
There are a couple of new print statements added that I'm guessing were for debugging. If so, they can be removed.

Reply:
Ohh, yeah! Sorry, I'll just clean those. Thanks.

Reply:
Done